.eslintrc

```diff
@@ -13,7 +13,7 @@
     // https://github.com/typescript-eslint/typescript-eslint/blob/5b0b3d9edbcb3ab588a34c431037d9deece30824/packages/eslint-plugin/docs/rules/indent.md#options
     "@typescript-eslint/indent": ["error", 2],
     "import/no-extraneous-dependencies": ["error", {
-        "devDependencies": ["**/*.spec.*", "src/setupTests.tsx"]
+        "devDependencies": ["**/*.spec.*", "src/setupTests.tsx", "src/test-helpers.tsx"]
```

Comment:
Why is this needed?

Reply:
Without it I get "2:1   error    'enzyme' should be listed in the project's dependencies, not devDependencies" on `/src/test-helpers.tsx`. I assume this is  because `ESLint` thinks this file is a prod file and not a test-only file, and this fixes that.